responders/tarpit/tarpit.go

```diff
@@ -141,3 +148,13 @@ func (r *Responder) ServeHTTP(w http.ResponseWriter, req *http.Request, _ caddyh
 		}
 	}
 }
+
+func (r *Responder) Validate() error {
+	if r.Config.Timeout <= 0 {
+		return fmt.Errorf("tarpit timeout must be greater than 0")
+	}
+	if r.Config.BytesPerSecond <= 10 {
+		return fmt.Errorf("tarpit bytes_per_second must be greater than 10 ")
```

Comment:
No need for Errorf since we're not formatting any strings, there's also a trailing space at the end

Reply:
Will fix! Thanks